### PR TITLE
Merge up-spec and up-core-api

### PR DESCRIPTION
**Pull request by stevenhartley**
the following change takes the 1.5.7 release of up-core-api and merges it with up-spec so we can have better traceability with the spec and protos.

#88

### CODE REVIEW

**Comment by stevenhartley** [comment]
@sophokles73 any objections to merge this PR and freeze the core-api project?

**Review by PLeVasseur — APPROVED** [review]
LGTM aside from the one thing I found which still pointed to `up-core-api`

File: up-core-api/uprotocol/uattributes.proto [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    // RPC Request messages MUST have this property set to a value > 0.
+    optional uint32 ttl = 6;
+    
+    // The service consumer's permission level as defined in
+    // https://github.com/eclipse-uprotocol/uprotocol-spec/blob/main/basics/permissions.adoc[Permissions].
+    //
+    // Publish, Notification and RPC Response messages MUST NOT have this property set to any value.
+    optional uint32 permission_level = 7;
+
+    // A https://github.com/eclipse-uprotocol/up-core-api/blob/main/uprotocol/ustatus.proto[code] indicating an error that has occurred
```
[the remark this answers is not in the document]
Comment (stevenhartley) [reply up-core-api/uprotocol/uattributes.proto]: Done

File: basics/upayload.adoc [hunk 2]
```diff
@@ -32,4 +32,4 @@ SPDX-License-Identifier: Apache-2.0
 == Overview 
 UPayload contains the actual message payload (either by value or reference) as well as metadata such as the format, size, etc... 
 
-NOTE: Please refer to https://github.com/eclipse-uprotocol/up-core-api/blob/main/uprotocol/upayload.proto[upayload.proto] for more details
+NOTE: Please refer tolink:../../up-core-api/uprotocol/upayload.proto[upayload.proto] for more details
```
Comment (sophokles73) [inline basics/upayload.adoc:35]: missing whitespace

File: up-l3/usubscription/v3/README.adoc [hunk 3]
```diff
@@ -37,7 +37,7 @@ Off the shelf broker and broker-less solutions, rely on specific Internet protoc
 uSubscriptions purpose is to explain _how_ the distributed publisher-subscriber architecture pattern can be implemented through a ubiquitous language and communication protocol across the distributed devices. We will define the interfaces, sequences, and state machines to be able to support the use cases mentioned above.
 
 
-NOTE: uSubscription interface is defined https://github.com/eclipse-uprotocol/up-core-api/blob/main/uprotocol/core/usubscription/v3/usubscription.proto[usubscription.proto] and follows link:../../../basics/error_model.adoc[uProtocol Error Model] for return codes.
+NOTE: uSubscription interface is definedlink:../../../up-core-api/uprotocol/core/usubscription/v3/usubscription.proto[usubscription.proto] and follows link:../../../basics/error_model.adoc[uProtocol Error Model] for return codes.
```
Comment (sophokles73) [inline up-l3/usubscription/v3/README.adoc:40]: missing whitespace

**Review by sophokles73 — CHANGES_REQUESTED** [review]

File: basics/upayload.adoc (hunk 2 above)
Comment (stevenhartley) [reply basics/upayload.adoc:35]: fixed

File: up-l3/usubscription/v3/README.adoc (hunk 3 above)
Comment (stevenhartley) [reply up-l3/usubscription/v3/README.adoc:40]: fixed

**Review by sophokles73 — APPROVED** [review]
LGTM

**Pull request merged**